looprpc/client.proto

```diff
@@ -1440,3 +1478,220 @@ message InstantOut {
     */
     string sweep_tx_id = 5;
 }
+
+message NewStaticAddressRequest {
+    /*
+    The client's public key for the 2-of-2 MuSig2 taproot static address.
+    */
+    bytes client_key = 1;
+}
+
+message NewStaticAddressResponse {
+    /*
+    The taproot static address.
+    */
+    string address = 1;
+
+    /*
+    The CSV expiry of the static address.
+    */
+    uint32 expiry = 2;
+}
+
+message ListUnspentDepositsRequest {
+    /*
+    The number of minimum confirmations a utxo must have to be listed.
+     */
+    int32 min_confs = 1;
+
+    /*
+    The number of maximum confirmations a utxo may have to be listed. A zero
+    value indicates that there is no maximum.
+    */
+    int32 max_confs = 2;
+}
+
+message ListUnspentDepositsResponse {
+    /*
+    A list of utxos behind the static address.
+    */
+    repeated Utxo utxos = 1;
+}
+
+message Utxo {
+    /*
+    The static address of the utxo.
+    */
+    string static_address = 1;
+
+    /*
+    The value of the unspent coin in satoshis.
+    */
+    int64 amount_sat = 2;
+
+    /*
+    The outpoint in the form txid:index.
+    */
+    string outpoint = 3;
+
+    /*
+    The number of confirmations for the Utxo.
+    */
+    int64 confirmations = 4;
```

Comment:
Maybe put `confirmation_height` instead of `confirmations` to make the response independent of the current height? In case somebody wants to save the response for future inspection, it would be nicer to have block heights here. Current tip height can be added as a separate field to ListUnspentDepositsResponse and the subtraction can be done on the client side.